centreon/www/include/configuration/configObject/contact/DB-Func.php

```diff
@@ -635,6 +635,9 @@ function insertContact($ret = array())
         $ret = $form->getSubmitValues();
     }
     $ret["contact_name"] = $centreon->checkIllegalChar($ret["contact_name"]);
+    if (isset($ret['contact_oreon']) && $ret['contact_oreon']['contact_oreon'] === '1') {
```

Comment:
also check for ['contact_oreon']['contact_oreon'] exists,

Here you check that 'contact_oreon' from $ret exists, but if 'contact_oreon' does not exists in $ret['contact_oreon'] you'll have an undefined array key